apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/Google.Cloud.Spanner.Data.IntegrationTests.csproj

```diff
@@ -10,7 +10,7 @@
     <NoWarn>1701;1702;1705;4014</NoWarn>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netcoreapp1.0|AnyCPU'">
-    <DefineConstants>TRACE;DEBUG;NETCOREAPP1_0;NETCOREAPP1_0</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;NETCOREAPP1_0</DefineConstants>
```

Comment:
I still suspect this whole PropertyGroup could be removed, but at least it's better now :) (And if VS would keep putting it back, I can see that's annoying.)